qa/workunits/rbd/cli_generic.sh

```diff
@@ -1468,6 +1493,178 @@ test_mirror_snapshot_schedule_recovery() {
     ceph osd pool rm rbd3 rbd3 --yes-i-really-really-mean-it
 }
 
+test_mirror_group_snapshot_schedule() {
+    echo "testing mirror group snapshot schedule..."
+    remove_images
+    ceph osd pool create rbd2 8
+    rbd pool init rbd2
+    rbd namespace create rbd2/ns1
+
+    rbd mirror pool enable rbd2 image
+    rbd mirror pool enable rbd2/ns1 image
+    rbd mirror pool peer add rbd2 cluster1
+
+    test "$(ceph rbd mirror group snapshot schedule list)" = "{}"
```

Comment:
why do commands begin with `ceph` here?

Reply:
This is because the test invokes the underlying manager command directly, bypassing the rbd CLI wrapper.

Reply:
ok but for normal cli execution it works with just `rbd mirror group snapshot schedule list` Is it ?